src/pages/HomePage/HomePage.js

```diff
@@ -8,6 +9,34 @@ import TextField from "@material-ui/core/TextField";
 import Button from "@material-ui/core/Button";
 
 const HomePage = () => {
+  const [state, setState] = useState({ email: "" });
+
+  const handleChange = (event) => {
+    setState({ email: event.target.value });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
```

Comment:
It's worth adding error checking and checking if the email is valid at this point